lava-provider/deploy.yml

```diff
@@ -0,0 +1,44 @@
+version: '2.0'
+endpoints:
+ <uniq_name_endpoint>: # unique name, for example "lava_dimokus:", and enter this name in line 24.
```

Comment:
let's just keep a working (real) endpoint name here so people can deploy it with the less modifications as well as less maintenance required (don't specify which lines need to be updated).

You can keep the comment that the endpoint name must be unique until https://github.com/akash-network/support/issues/108 is resolved